### ISSUE TITLE
Aspect Ratio argument for renpy.movie_cutscene()

### ISSUE THREAD

**Issue by RaXorX** [renpy/renpy#4329]
Feature Request - 

Is there a possibility that renpy.movie_cutscene() function has the following argument - aspect_ratio - to keep the video's aspect ratio intact on android (and just in case - not that I have tested but also for desktop - or any device all in all, matter of factly)?

Yes I can see that currently it means fullscreen but videos don't really look the same when their aspect ratio changes. Or at best there could be something of the sort - fill_width or fill_height - which could be a boolean value to determine when a video should be filled horizontally/vertically or not.

**Comment by renpytom** [comment]
I just tested this, and and renpy.movie_cutscene preserves aspect ratio by default, including on Android. Are you seeing something different?

**Comment by RaXorX** [comment]
> I just tested this, and and renpy.movie_cutscene preserves aspect ratio by default, including on Android. Are you seeing something different?

Yes, for me it changes the aspect ratio of the video on my phone. Not sure why, I can try to test it on another device and see what the result is.
